helms/knative/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+name: knative
```

Comment:
Maybe, we might name it with suffix/prefix `legion`?

Reply:
Why? It's a clean Knative deployment without even one legion component or extension.

Reply:
@mcdoker18 but why we are copying their helm chart into our repo in this case?

Reply:
ok

Reply:
can we use knative chart from public helm repo or there are some incompatible modifications in the story?